meta-client/src/main/java/org/apache/helix/metaclient/recipes/leaderelection/LeaderElectionClient.java

```diff
@@ -288,7 +289,8 @@ public List<String> getParticipants(String leaderPath) {
    * @return A boolean value indicating if registration is success.
    */
   public boolean subscribeLeadershipChanges(String leaderPath, LeaderElectionListenerInterface listener) {
-    //TODO: add converter class for LeaderElectionListenerInterface
+    _metaClient.subscribeDataChange(leaderPath + "/LEADER", new LeaderElectionListenerInterfaceAdapter(listener),
+        false);
```

Comment:
Question: does this mean this is non-persist watcher or persist watcher?

If it is persist watch, if the ephemeral node is gone, then is the watcher still valid and receive new leader acquired notification?

Reply:
It is persist watcher. If the ephemeral node is gone, then is the watcher still valid and receive new leader acquired notification.

Reply:
Let's not hard code "LEADER"